modules/node_modules/@ncigdc/containers/CaseAggregations.js

```diff
@@ -34,14 +46,94 @@ export type TProps = {
        id: string,
      |},
    |}>,
- },
+  },
   setAutocomplete: Function,
   theme: Object,
   suggestions: Array<Object>,
+
+  userSelectedFacets: Array<{|
+    description: String,
+    doc_type: String,
+    field: String,
+    full: String,
+    type: "id" | "string" | "long",
+    |}>,
+  handleSelectFacet: Function,
+  handleResetFacets: Function,
+  presetFacetFields: Array<String>,
+  shouldShowFacetSelection: Boolean,
+  facetExclusionTest: Function,
+};
+
+const storageKey = 'RepositoryCaseAggregations.userSelectedFacets';
+const presetFacetFields = [
+  'project.primary_site',
```

Comment:
case_id and submitter_id are also preset facets - if the preset facets were also an array mapped over when being displayed instead of maintaining this separate array might be better